sdk/tests/conformance/glsl/misc/shaders-with-name-conflicts.html

```diff
@@ -56,25 +60,27 @@
 Section 4.3.4
 
 The uniform qualifier is used to declare global variables
+---
 
-QED: If both uniforms and attributes are in the global namespace they
-conflict across shaders
+that an attribute in a vertex shader would conflict with a uniform of
+the same name in the fragment shader.
 
-Note: This was brought up on the OpenGL ES working group and confirmed
-that the spec required these conflicts to fail to link.
+GLSL ES 3.00 later confirmed that pipeline stages have distinct
+namespaces. The OpenGL ES working group confirmed in discussions
+https://github.com/KhronosGroup/WebGL/issues/3201 that this was the
+intended behavior for GLSL ES 1.00.
 
-Though most drivers allow this to work, some drivers to do no therefore
-WebGL implementation must enforce this restriction to provide consistent
-behavior.
+For this reason, this test asserts that such usage does not generate a
+conflict.
```

Comment:
This diff seems sorta garbled?

Reply:
I left some history in the comment. The changes are intentional as is.

Reply:
I find this confusing to read, but I see it now.
The section is really just too big to be inlined like this. Consider

```
This test previously concluded that an attribute in a vertex shader
would conflict with a uniform of conflict across shaders the same name
in the fragment shader, based on the following sections of the GLSL
ES 1.0.17 specification:

---

Section 4.2.6
[...]
With the exception of uniform declarations, vertex and fragment shaders
have separate name spaces.  Functions and global variables declared in a
vertex shader cannot be referenced by a fragment shader and vice versa.
Uniforms have a single name space.  Uniforms declared with the same name
must have matching types and precisions.

Section 4.3.3
Attribute variables are required to have global scope

Section 4.3.4
The uniform qualifier is used to declare global variables

---

GLSL ES 3.00 later confirmed...
```

Reply:
Thanks, good suggestion. Have restructured it in this way.